content/en/docs/reference/components/vault.md

```diff
@@ -4,106 +4,56 @@ linktitle: Vault
-computing environments such as cloud service providers and Git hosts
-offer decentralized solutions with distinct advantages in service
-reliability and costs over the traditional solutions.
+## Todo: issue-2825
 
-However, one issue with using cloud services, distributed storage, and
-remote repositories is the lack of trusted networks, vetted hardware,
-and other closely observed security measures practiced in locally-hosted
-infrastructure. For the sake of convenience, users often store sensitive
-information like authentication credentials in open, public
-repositories, exposed to potential malicious activity.
+* installation of an external Vault instance
+* accessing secrets
+* secret rotation
 
-[Hashicorp *Vault*](https://www.vaultproject.io/) is one tool that
-centralizes the management of *secrets*: resources that provide
-authentication to your computing environment such as tokens, keys,
-passwords, and certificates.
+----
 
-Jenkins X handles security and authentication
-resources through the integration of Vault. Users can deploy Vault to
-securely store and manage all aspects of their development platform.
-
-Jenkins X installs and configures Vault for your
-cluster by default through the cluster creation process.
-
-## Vault features
-
-Vault is a tool for accessing and storing user secrets. It manages the
-complexity of secure resource access:
-
-- Storing secrets - Vault places secrets in an encrypted format in a
-remote storage bucket.
-
-- Secret creation and deletion - Vault creates secrets for dynamic
-access to storage buckets, ephemeral access that are
-created/destroyed as needed for temporary data access, and
-generating keys for database authentication.
-
-- Encrypting data - Vault stores secrets in a remote storage bucket in
-secure directories using strong encryption.
-
-Jenkins X interacts with Vault via the `jx`
-command line program. There are commands for creating, deleting, and
-managing secrets and vaults.
-
-Jenkins X uses Vault to store all Jenkins X
-secrets, such as the GitHub personal access token generated for the
-pipeline bot when [creating a Jenkins X
-cluster](/docs/getting-started/setup/boot/). It also stores
-any GitOps secrets, such as passwords for storage buckets, and keys for
-secure server access.
-
-Secrets can be retrieved by the pipeline or via command-line if logged
-into the account associated with the kubernetes service as well as any
-secrets stored in the `jx` namespace for the pipeline.
-
-Vaults are provisioned in kubernetes using `vault-operator`, an
-open-source Kubernetes controller installed when Vault is configured
-during cluster creation and Jenkins X installation on the cluster.
-
-# Using Vault on the CLI
+## Using Vault on the CLI
 
 First you need to download an install the [safe](https://github.com/starkandwayne/safe) CLI for Vault.
 
 Once you have installed [safe](https://github.com/starkandwayne/safe) you can run:
```

Comment:
Why safe? Why not the bank-vault cli, since we use bank vault anyways? Is safe better?